develop/develop/settings.py

```diff
@@ -99,6 +101,20 @@
 else:
     DATABASES['default'] = dj_database_url.config(conn_max_age=600, ssl_require=True, default=os.environ.get('DATABASE_URL'))
 
+    
+# Cache backend is optional, but recommended to speed up user agent parsing
+CACHES = {
```

Comment:
We shouldn't have caching in the project while developing as it can often lead to data issues.